### PR TITLE
Enhancement/booking

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

**Comment by ayyubibrahimi** [comment]
@pckhoi I was tipped off to [this ](http://www.lcle.state.la.us/programs/lavns_roster.asp) hub for databases which each contain booking data for all parishes in Louisiana, except for four. I wanted to get your feedback on this scraping script before I write one for each parish. 

Most parishes show that their last update was on 9/26, therefore, I assume that we'll only need to scrape once a month.

File: clean/plaquemines_so_vine_booking.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        dob = cells[1].text
+        race = cells[2].text
+        gender = cells[3].text
+        arrest_date = cells[4].text
+        inmate = {'name': name, 'birth_date': dob, 'race': race, 'gender': gender, 'arrest_date': arrest_date}
+        inmates_dict.append(inmate)
+
+
+def clean():
+    df = pd.DataFrame.from_dict(inmates_dict)\
```
Comment (pckhoi) [inline clean/plaquemines_so_vine_booking.py:26]: Not sure how this is different but for constructing a dataframe from a list of dicts I usually use `DataFrame.from_records`.

File: clean/plaquemines_so_vine_booking.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        .pipe(clean_column_names)
+    return df
+
+
+if __name__ == '__main__':
+    df = clean()
+    df.to_csv(data_file_path('clean/plaquemines_so_booking_log_10_21_2021.csv'), index=False)
```
[the remark this answers is not in the document]
Comment (pckhoi) [reply clean/plaquemines_so_vine_booking.py]: Running things with `make` only makes sense if there are dependencies and you want changes to cascade. Doesn't seem like this file is connected to any other script.

So it sounds like you might want to just put the current date there (i.e. `datetime.now()`). It'll work regardless of whether you run it manually or in a job queue (i.e. Github actions).

Another option is to simply do away with the date in the filename and simply commit it directly to Wrgl, given that you are probably only interested in how the data change over time anyway.

File: clean/plaquemines_so_vine_booking.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                race = cells[2].text
+                sex = cells[3].text
+                arrest_date = cells[4].text
+                inmate = {'name': name, 'birth_date': dob, 'race': race, 'sex': sex, 'arrest_date': arrest_date}
+                inmates.append(inmate)
+    except Exception as e:
+        return e
+
+    df = pd.DataFrame.from_records(inmates)
+    df.to_csv(data_file_path('scraped/plaquemines_so_booking_log_10_21_2021_to_10_26_2021.csv'), index=False)
```
Comment (pckhoi) [inline clean/plaquemines_so_vine_booking.py:28]: There's no need to output to csv if no other script is ever going to read this data.

File: clean/plaquemines_so_vine_booking.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (pckhoi) [reply clean/plaquemines_so_vine_booking.py]: You don't need a Mac, just having WLS is enough, there are binaries for Linux. However, you need to wait a few weeks. I'm finalizing the new open-sourced Wrgl and rebranding old website as Wrgl Hub. With that the binary, API everything will change.

**Pull request closed without merging**